docs/docs/how-to/cloud/enable-image-cdn.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Enable Image CDN"
+description: "Learn how to accelerate builds locally and in Gatsby Cloud by offloading image processing to Gatsby's Image CDN"
+---
+
+Before continuing, make sure you understand [What is Image CDN](../../../reference/cloud/image-cdn/). Also, note that Image CDN only works on sites hosted on Gatsby cloud and doesn't work with other [hosting integrations](https://support.gatsbyjs.com/hc/en-us/articles/1500000907821-Hosting-and-Data-Source-Integrations).
+
+## 1. Enable Image CDN
+
+Image CDN is enabled per site on Gatsby Cloud. Go to Site Settings → Images → Enable Image CDN.
+
+![How to enable Gatsby's Image CDN](../../images/image-cdn-enable.png)
+
+## 2. Upgrade packages
+
+Image CDN requires the latest Gatsby 4 version. Upgrade core dependencies:
+
+```shell
+npm install gatsby@latest gatsby-plugin-image@latest gatsby-plugin-sharp@latest gatsby-transformer-sharp@latest
+```
+
+ or
+
+```shell
+yarn add gatsby@latest gatsby-plugin-image@latest gatsby-plugin-sharp@latest gatsby-transformer-sharp@latest
+```
+(note that on later npm versions you may need to install with the `--legacy-peer-deps` npm flag)
+
+## 3. Upgrade source plugins
+
+Upgrade supported source plugins to the latest tag. Source plugins currently supported:
+
+```shell
+gatsby-source-contentful@^7.8.0
+gatsby-source-wordpress@^6.10.0
+gatsby-source-sanity@^7.4.0
+```
+
+## 4. Migrate to gatsbyImage field
```

Comment:
This is really vague and missing details and code snippets.

I also think that sentences like "gatsbyImage replaces gatsbImageData" are just incorrect. They do not have feature parity so the former can't replace the latter.

> gatsbyImage is compatible whether or not you’re using Image CDN and is the new field for using Gatsby-powered images

That's also really vague, the first half of the sentence means that we fallback to normal image generation if Image CDN is not enabled, but that's not conveyed here at all. Also, it's effectively not the new field for gatsby-powered images as said above.